### ISSUE TITLE
Typo in jetty.sh

### ISSUE THREAD

**Issue by DavidShepherdson** [eclipse/jetty.project#2762]
Running `jetty.sh` in 9.4.11.v20180605, I see this error if I set `JETTY_START_TIMEOUT`:

    jetty.sh: line 388: [: missing `]'

This *appears* to be harmless (in that it doesn't prevent the script from starting Jetty up), but I'm not sure whether it is actually using my timeout value or not (and in any case there probably shouldn't be an error reported like this :-)).

Looking at line 388 of that script, it has:

    if [ -z "$JETTY_START_TIMEOUT"]

In other words, it looks like there's a space missing before that closing `]`.

**Comment by sbordet** [comment]
Thanks for reporting this. For such a simple fix, would you like to give it a go and issue a pull request?

**Comment by DavidShepherdson** [comment]
Thanks @sbordet -- I've had a go at doing just that. Please let me know if there are any problems.

**Comment by sbordet** [comment]
Thanks!
